tests/integration/src/exec_vm.rs

```diff
@@ -1,10 +1,12 @@
+#![allow(unused)]
```

Comment:
*Nit:* Is this actually needed? I prefer to only disable warnings module-wide if it is especially noisy and unhelpful; otherwise it's usually easier to just decorate specific items with this attribute instead.

Reply:
Not really. I agree that decorating a specific item is a better approach.

Reply:
Done.